index.html

```diff
@@ -0,0 +1,177 @@
+        </div>    
+        <section id="header-about-section"> 
+            <h2 class="red">BEATZ Station Music streaming</h2>
+            <div>
+                <p id="about-main-content" class="black">
+                    Lorem ipsum dolor sit, amet consectetur adipisicing elit. 
+                    Aliquid veritatis reprehenderit voluptatem cum libero omnis, quisquam quas suscipit
+                    ratione quibusdam incidunt non ducimus dicta minima consectetur vitae inventore facilis nulla?
+                </p>
+            </div>
+            <div class="visit-link">
+                <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit.</p>
+                <p><strong>ccsummit2022@ccworldwide.org</strong></p>
+            </div>
+        </section>
+    </header>
+    <div id="bodi-homepage-section">
+        <div class="latest-songs">
+            <h2 class="gray">New Releases</h2>
+            <hr class="line-border">
+        </div>
+        <div class="songs-list ">
+            <div class="song">
+                <div><img src="media/music.svg" alt="Disk" width="60" height="60"></div>
+                <div><h3 class="red">Play</h3></div>
+                <div><p>Listen to the speakers from various countries about the messages of sharing and opening</p></div>
+            </div>
+            <div class="song">
+                <div><img src="media/music.svg" alt="Disk" width="60" height="60"></div>
+                <div><h3 class="red">Play</h3></div>
+                <div><p>Listen to the speakers from various countries about the messages of sharing and opening</p></div>
+            </div>
+            <div class="song">
+                <div><img src="media/music.svg" alt="Disk" width="60" height="60"></div>
+                <div><h3 class="red">Play</h3></div>
+                <div><p>Listen to the speakers from various countries about the messages of sharing and opening</p></div>
+            </div>
+            <div class="song">
+                <div><img src="media/music.svg" alt="Disk" width="60" height="60"></div>
+                <div><h3 class="red">Play</h3></div>
+                <div><p>Listen to the speakers from various countries about the messages of sharing and opening</p></div>
+            </div>
+            <div class="song">
+                <div><img src="media/music.svg" alt="Disk" width="60" height="60"></div>
+                <div><h3 class="red">Play</h3></div>
+                <div><p>Listen to the speakers from various countries about the messages of sharing and opening</p></div>
+            </div>
+            <button id="selectsong" type="button">Jump on the Beat</button>
+        </div>
+        <div class="see-more"><p>SEE ALL EVENT</p></div>
+    </div>
+    <section id="bodi-about-section"> 
+     <div class="about-bodi"> 
+        <div>
+            <div class="summit">
+                <h2>Creative commons global summit 2022 logo</h2>
+                <hr>
+            </div>
```

Comment:
- [ ] OPTIONAL: Please consider centring the logo in `About` page (Mobile view): 

<img width="449" alt="Screenshot 2022-03-11 at 18 34 08" src="https://user-images.githubusercontent.com/77038610/157887679-c5763aab-f6ec-4669-a3f4-9e107d30785c.png">